python/cuml/svm/linear_svc.py

```diff
@@ -0,0 +1,187 @@
+    penalty : {{'l1', 'l2'}} (default = '{LinearSVM_defaults.penalty}')
+        The regularization term of the target function.
+    loss : {LinearSVC.REGISTERED_LOSSES} (default = 'squared_hinge')
+        The loss term of the target function.
+    fit_intercept : {LinearSVM_defaults.fit_intercept.__class__.__name__ \
+            } (default = {LinearSVM_defaults.fit_intercept})
+        Whether to fit the bias term. Set to False if you expect that the
+        data is already centered.
+    penalized_intercept : { \
+            LinearSVM_defaults.penalized_intercept.__class__.__name__ \
+            } (default = {LinearSVM_defaults.penalized_intercept})
+        When true, the bias term is treated the same way as other features;
+        i.e. it's penalized by the regularization term of the target function.
+        Enabling this feature forces an extra copying the input data X.
+    max_iter : {LinearSVM_defaults.max_iter.__class__.__name__ \
+            } (default = {LinearSVM_defaults.max_iter})
+        Maximum number of iterations for the underlying solver.
+    linesearch_max_iter : { \
+            LinearSVM_defaults.linesearch_max_iter.__class__.__name__ \
+            } (default = {LinearSVM_defaults.linesearch_max_iter})
+        Maximum number of linesearch (inner loop) iterations for
+        the underlying (QN) solver.
+    lbfgs_memory : { \
+            LinearSVM_defaults.lbfgs_memory.__class__.__name__ \
+            } (default = {LinearSVM_defaults.lbfgs_memory})
+        Number of vectors approximating the hessian for the underlying QN
+        solver (l-bfgs).
+    verbose : int or boolean, default=False
+        Sets logging level. It must be one of `cuml.common.logger.level_*`.
+        See :ref:`verbosity-levels` for more info.
+    C : {LinearSVM_defaults.C.__class__.__name__ \
+            } (default = {LinearSVM_defaults.C})
+        The constant scaling factor of the loss term in the target formula
+          `F(X, y) = penalty(X) + C * loss(X, y)`.
+    grad_tol : {LinearSVM_defaults.grad_tol.__class__.__name__ \
+            } (default = {LinearSVM_defaults.grad_tol})
+        The threshold on the gradient for the underlying QN solver.
+    change_tol : {LinearSVM_defaults.change_tol.__class__.__name__ \
+            } (default = {LinearSVM_defaults.change_tol})
+        The threshold on the function change for the underlying QN solver.
+    tol : Optional[float] (default = None)
+        Tolerance for the stopping criterion.
+        This is a helper transient parameter that, when present, sets both
+        `grad_tol` and `change_tol` to the same value. When any of the two
+        `***_tol` parameters are passed as well, they take the precedence.
+    probability: {LinearSVM_defaults.probability.__class__.__name__ \
+            } (default = {LinearSVM_defaults.probability})
+        Enable or disable probability estimates.
+    multiclass_strategy : {{currently, only 'ovr'}} (default = 'ovr')
```

Comment:
Not sure what is the right argument name, maybe @dantegd can give his suggestion:
- `multiclass_strategy` is used by `cuml.svm.SVC`
- `multi_class` is used by `sklearn.svm.LinearSVC`
- In practice we do not have a  choice, currently only `'ovr'` is implemented.

Reply:
NB from our teams discussion: a potential solution would be to add the same ephemeral parameter as for `tol`. So that our LinearSVC would support both for compatibility with sklearn but store only one.

Reply:
I am slightly in favor of using the same arg name as sklearn. An ephemeral arg `multiclass_strategy` to be consistent with `cuml.svm.SVC` could be added, but as long as we have only one method I do not think it is important.